### ISSUE TITLE
Export .vcf and .ics

### ISSUE THREAD

**Issue by charsi** [39aldo39/DecSyncCC#12]
Just a feature suggestion. Maybe for greater app interoperability DecSync should also save compiled vcf and ics files ?

**Comment by 39aldo39** [comment]
DecSync CC already integrates with the Android contacts and calendars, which gives great interoperability. Could you clarify a use case a bit more?

**Comment by charsi** [comment]
I was thinking for the desktop if i have access to the ics file i could point thunderbird to that to see the calendar. Saves having to rely on a plugin to read DecSync format.
Same for contacts. Many apps already out there  that understand vcf.

**Comment by 39aldo39** [comment]
I see, support for Thunderbird is indeed nice, but DecSync CC is just the Android app ;). See also 39aldo39/DecSync#9 (closing this for that).

**Comment by charsi** [comment]
Yes DecSync CC is just the android app. But I still think it would be nice to have the option to save vcf and ics in parallel. This way users don't have to implement support for DecSync format on other platforms. 

As you mentioned in the other ticket support in Radicale would be one way to go. But then it defeats the 'serverless' solution that DecSync currently offers. 

Btw.. I've asked for DecSync CC to be added to the awesome-selfhosted list here -https://github.com/Kickball/awesome-selfhosted/issues/1691

**Comment by 39aldo39** [comment]
In order to make DecSync more available for other applications, I think that a Radicale plugin is better than vcf and ics files:
* If the vcf and ics files are modified, it is hard to determine what exactly changed, and what changes came from other apps. I am worried it can lead to some sync conflicts.
* You can install Radicale on your own computer and point the CalDAV/CardDAV clients to the localhost, no server required. I actually did this before I made DecSync, but DecSync has the advantage that it doesn't require sync intervals between devices, there is no master and you can sync other types of data.

Thank you for asking DecSync to be added to the awesome-selfhosted list! It helps with making it more known.

**Comment by charsi** [comment]
> If the vcf and ics files are modified, it is hard to determine what exactly changed, and what changes came from other apps.
Hmm DScc shouldn't have to rely on vcf, ics for that info. The decsync db should still exist and just save these formats 'in addition' whenever there are changes. 

Running Radicale locally is a good enough solution. But it is still yet another thing that I have to have running. Much prefer a strictly file based approach if it can be achieved.
